openshift/templates/orgbook-issuer-controller/orgbook-issuer-controller-build.yaml

```diff
@@ -0,0 +1,125 @@
+    apiVersion: v1
+    metadata:
+      name: ${NAME}${SUFFIX}
+      labels:
+        name: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        app.kubernetes.io/part-of: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: ${NAME}${SUFFIX}
+      labels:
+        name: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        app.kubernetes.io/part-of: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+    spec:
+      runPolicy: Serial
+      source:
+        type: Git
+        git:
+          ref: ${GIT_REF}
+          uri: ${GIT_REPO_URL}
+        contextDir: ${SOURCE_CONTEXT_DIR}
+      strategy:
+        type: Docker
+        dockerStrategy:
+          dockerfilePath: ${DOCKER_FILE_PATH}
+          from:
+            kind: DockerImage
+            name: ${SOURCE_IMAGE_NAME}
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}${SUFFIX}:${OUTPUT_IMAGE_TAG}
+      resources:
+        requests:
+          cpu: ${CPU_REQUEST}
+          memory: ${MEMORY_REQUEST}
+        limits:
+          cpu: ${CPU_LIMIT}
+          memory: ${MEMORY_LIMIT}
+      triggers:
+        - type: ConfigChange
+parameters:
+  - name: NAME
+    displayName: Name
+    description: The name assigned to all of the objects defined in this template.
+    required: true
+    value: controller
+  - name: SUFFIX
+    displayName: Suffix
+    description: A name suffix used for all objects
+    required: false
+    value: -lcrb
+  - name: APP_NAME
+    displayName: App Name
+    description: Used to group components together in the OpenShift console.
+    required: true
+    value: issuer
+  - name: APP_GROUP
+    displayName: App Group
+    description: The name assigned to all of the deployments in this project.
+    required: true
+    value: lcrb-orgbook-issuer
+  - name: GIT_REPO_URL
+    displayName: Git Repo URL
+    description: The URL to your GIT repo.
+    required: true
+    value: https://github.com/bcgov/aries-vcr-issuer-controller.git
+  - name: GIT_REF
+    displayName: Git Reference
+    description: The git reference or branch.
+    required: true
+    value: master
+  - name: SOURCE_CONTEXT_DIR
+    displayName: Source Context Directory
+    description: The source context directory.
+    required: false
+    value: issuer_controller
+  - name: OUTPUT_IMAGE_TAG
+    displayName: Output Image Tag
+    description: The tag given to the built image.
+    required: true
+    value: latest
+  - name: DOCKER_FILE_PATH
+    displayName: Docker File Path
+    description: The path to the docker file.
+    required: true
+    value: Dockerfile
+  - name: SOURCE_IMAGE_NAME
+    displayName: Source Image Name
+    description: The name of the Docker image to be used for the build. Overrides the FROM instruction in the Dockerfile.
+    required: false
+    value: artifacts.developer.gov.bc.ca/docker-remote/bcgovimages/von-image:py36-1.16-0
```

Comment:
Does the controller need to be based on `von-image` or can we use a generic python container like we're doing now with the OrgBook API?

Reply:
We might be able to switch to a generic Python image, I need to double check. I believe looked into this a while back and there was something that prevented us from uplifting it, but something prevented us from moving forward. Will review and get back to you as I stand-up the service.